doc/release/1.14.0-notes.rst

```diff
@@ -265,26 +265,26 @@ In summary, the major changes are:
   * Float arrays printed in scientific notation no longer use fixed-precision,
     and now instead show the shortest unique representation.
   * The ``str`` of floating-point scalars is no longer truncated in python2.
- 
+
 * For other data types:
 
   * Non-finite complex scalars print like ``nanj`` instead of ``nan*j``.
   * ``NaT`` values in datetime arrays are now properly aligned.
   * Arrays and scalars of ``np.void`` datatype are now printed using hex
     notation.
-    
+
 * For line-wrapping:
 
   * The "dtype" part of ndarray reprs will now be printed on the next line
     if there isn't space on the last line of array output.
   * The ``linewidth`` format option is now always respected.
     The `repr` or `str` of an array will never exceed this, unless a single
     element is too wide.
-  * All but the last line of array strings will contain the same number of
-    elements.
```

Comment:
This is only a change in extremely contrived scenarios (custom separators), so probably doesn't belong here.